triplet/main.py

```diff
@@ -230,6 +238,31 @@ def train(epoch):
                 100. * batch_idx / len(train_loader), loss.data[0]))
 
 
+def test(epoch):
+    model.eval()
+    scores, distances = [], []
+    for batch_idx, (data_a, data_p, label) in enumerate(test_loader):
+        if args.cuda:
+            data_a, data_p = data_a.cuda(), data_p.cuda()
+        data_a, data_p, labels = Variable(data_a, volatile=True), \
+                                 Variable(data_p, volatile=True), Variable(label)
+        out_a, out_p = model(data_a), model(data_p)
+        dists = torch.sqrt(torch.sum((out_a - out_p) ** 2, 1))  # euclidean distance
+        distances.extend(dists)
+        scores.extend(label.numpy())
+        if batch_idx % args.log_interval == 0:
+            print('Test Epoch: {} [{}/{} ({:.0f}%)]'.format(
+                epoch, batch_idx * len(data_a), len(test_loader.dataset),
+                100. * batch_idx / len(test_loader)))
+
+        if batch_idx > 70:
+            break
```

Comment:
@apaszke I had to hack this otherwise I got the follwing error:
```
assert total % total_sizes == 0, "Can't make sizes have exactly %d elements" % total
AssertionError: Can't make sizes have exactly 829440 elements
```